tests/foreman/ui/test_activationkey.py

```diff
@@ -102,10 +102,10 @@ def test_positive_end_to_end_crud(session, module_org):
         # Update activation key with new name
         session.activationkey.update(name, {'details.name': new_name})
         assert session.activationkey.search(new_name)[0]['Name'] == new_name
-        assert not session.activationkey.search(name)
+        assert session.activationkey.search(name)[0]['Name'] != name
         # Delete activation key
         session.activationkey.delete(new_name)
-        assert not session.activationkey.search(new_name)
+        assert session.activationkey.search(name)[0]['Name'] != new_name
```

Comment:
Are you sure you want to search by `name` and not `new_name`?

Reply:
copy paste issue